lib/_stream_readable.js

```diff
@@ -381,7 +382,7 @@ function howMuchToRead(n, state) {
 // You can override either this method, or the async _read(n) below.
 Readable.prototype.read = function(n) {
   debug('read', n);
-  n = parseInt(n, 10);
```

Comment:
Should we not just validate the input instead to be either `undefined` or an integer?

Reply:
@bridgear That seems like an unnecessary breaking change to me? This PR is semver-major too, but it’s less impactful, imo